### PR TITLE
Added form integrity

### PR DESCRIPTION
**Pull request by traubisoda**

### CODE REVIEW

File: src/index.vue [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
     <slot></slot>
   </component>
 </template>
 
 <script>
 import Flatpickr from 'flatpickr'
-import SingleInput from './SingleInput'
-import WrapperInput from './WrapperInput'
+import SingleInput from './SingleInput.vue'
+import WrapperInput from './WrapperInput.vue'
```
Comment (fundon) [inline src/index.vue:11]: Is `.vue` required?

File: src/index.vue [hunk 2]
```diff
@@ -24,6 +24,7 @@ export default {
 
   props: {
     alignment: String,
+    inputName: String,
```
Comment (fundon) [inline src/index.vue:27]: `formName`?

**Comment by fundon** [comment]
Thank you.

File: src/index.vue (hunk 1 above)
Comment (ezerw) [reply src/index.vue:11]: I had the same issue, .vue is required

**Comment by fundon** [comment]
Re-open it if you want. Thanks.

**Pull request closed without merging**